### ISSUE TITLE
Why do the images I generated from different viewpoints look gray?

### ISSUE THREAD

**Issue by Dumeowmeow** [SUDO-AI-3D/zero123plus#96]
like this:
<img width="386" alt="1726303240487" src="https://github.com/user-attachments/assets/94da8cd4-ceda-4dd7-80d3-90e0544a37cf">

**Comment by eliphatfs** [comment]
What code are you using?

**Comment by Dumeowmeow** [comment]
> What code are you using?

The code I ran is as follows：
`import torch
import requests
from PIL import Image
from diffusers import DiffusionPipeline, EulerAncestralDiscreteScheduler

# Load the pipeline
pipeline = DiffusionPipeline.from_pretrained(
    "sudo-ai/zero123plus-v1.1", custom_pipeline="sudo-ai/zero123plus-pipeline",
    torch_dtype=torch.float16
)

# Feel free to tune the scheduler!
# `timestep_spacing` parameter is not supported in older versions of `diffusers`
# so there may be performance degradations
# We recommend using `diffusers==0.20.2`
pipeline.scheduler = EulerAncestralDiscreteScheduler.from_config(
    pipeline.scheduler.config, timestep_spacing='trailing'
)
pipeline.to('cuda:0')

# Download an example image.
cond = Image.open(requests.get("https://d.skis.ltd/nrp/sample-data/lysol.png", stream=True).raw)

# Run the pipeline!
result = pipeline(cond, num_inference_steps=75).images[0]
# for general real and synthetic images of general objects
# usually it is enough to have around 28 inference steps
# for images with delicate details like faces (real or anime)
# you may need 75-100 steps for the details to construct

import rembg
result = rembg.remove(result)
result.show()
result.save("output.png")`

**Comment by Dumeowmeow** [comment]
> What code are you using?
The code I ran is as follows:
`import torch
import requests
from PIL import Image
from diffusers import DiffusionPipeline, EulerAncestralDiscreteScheduler
pipeline = DiffusionPipeline.from_pretrained(
    "sudo-ai/zero123plus-v1.1", custom_pipeline="sudo-ai/zero123plus-pipeline",
    torch_dtype=torch.float16
)

pipeline.scheduler = EulerAncestralDiscreteScheduler.from_config(
    pipeline.scheduler.config, timestep_spacing='trailing'
)
pipeline.to('cuda:0')

cond = Image.open(requests.get("https://d.skis.ltd/nrp/sample-data/lysol.png", stream=True).raw)

result = pipeline(cond, num_inference_steps=75).images[0]
import rembg
result = rembg.remove(result)
result.show()
result.save("output.png")`

**Comment by Dumeowmeow** [comment]
> What code are you using?

The code I ran is as follows:
<img width="874" alt="1726378650336" src="https://github.com/user-attachments/assets/1aba873f-6c3a-48a4-97e3-980f58390f19">

**Comment by eliphatfs** [comment]
Can you provide the code as a formatted code block in the comment so i may try to reproduce the behavior on my side?

**Comment by Dumeowmeow** [comment]
> Can you provide the code as a formatted code block in the comment so i may try to reproduce the behavior on my side?

Thank you!Here is the code I used:
`import torch
import requests
from PIL import Image
from diffusers import DiffusionPipeline, EulerAncestralDiscreteScheduler

pipeline = DiffusionPipeline.from_pretrained(
    "sudo-ai/zero123plus-v1.1", custom_pipeline="sudo-ai/zero123plus-pipeline",
    torch_dtype=torch.float16
)

pipeline.scheduler = EulerAncestralDiscreteScheduler.from_config(
    pipeline.scheduler.config, timestep_spacing='trailing'
)
pipeline.to('cuda:0')

cond = Image.open(requests.get("https://d.skis.ltd/nrp/sample-data/lysol.png", stream=True).raw)

result = pipeline(cond, num_inference_steps=75).images[0]

import rembg
result = rembg.remove(result)
result.show()
result.save("output.png")`

**Comment by Dumeowmeow** [comment]
> Can you provide the code as a formatted code block in the comment so i may try to reproduce the behavior on my side?

Thank you!Here is the code I used:
`import torch
import requests
from PIL import Image
from diffusers import DiffusionPipeline, EulerAncestralDiscreteScheduler

pipeline = DiffusionPipeline.from_pretrained(
    "sudo-ai/zero123plus-v1.1", custom_pipeline="sudo-ai/zero123plus-pipeline",
    torch_dtype=torch.float16
)

pipeline.scheduler = EulerAncestralDiscreteScheduler.from_config(
    pipeline.scheduler.config, timestep_spacing='trailing'
)
pipeline.to('cuda:0')

cond = Image.open(requests.get("https://d.skis.ltd/nrp/sample-data/lysol.png", stream=True).raw)

result = pipeline(cond, num_inference_steps=75).images[0]

import rembg
result = rembg.remove(result)
result.show()
result.save("output.png")`

**Comment by Dumeowmeow** [comment]
Thank you！Here is the code I used, actually, the only difference between this and img_to_mv.py is that the background is removed at the end.

import torch
import requests
from PIL import Image
from diffusers import DiffusionPipeline, EulerAncestralDiscreteScheduler

pipeline = DiffusionPipeline.from_pretrained(
    "sudo-ai/zero123plus-v1.1", custom_pipeline="sudo-ai/zero123plus-pipeline",
    torch_dtype=torch.float16
)

pipeline.scheduler = EulerAncestralDiscreteScheduler.from_config(
    pipeline.scheduler.config, timestep_spacing='trailing'
)
pipeline.to('cuda:0')

cond = Image.open(requests.get("https://d.skis.ltd/nrp/sample-data/lysol.png", stream=True).raw)

result = pipeline(cond, num_inference_steps=75).images[0]

import rembg
result = rembg.remove(result)
result.show()
result.save("output.png")

**Comment by Dumeowmeow** [comment]
> Can you provide the code as a formatted code block in the comment so i may try to reproduce the behavior on my side?

Thank you！Here is the code I used, actually, the only difference between this and img_to_mv.py is that the background is removed at the end.

import torch
import requests
from PIL import Image
from diffusers import DiffusionPipeline, EulerAncestralDiscreteScheduler

pipeline = DiffusionPipeline.from_pretrained(
"sudo-ai/zero123plus-v1.1", custom_pipeline="sudo-ai/zero123plus-pipeline",
torch_dtype=torch.float16
)

pipeline.scheduler = EulerAncestralDiscreteScheduler.from_config(
pipeline.scheduler.config, timestep_spacing='trailing'
)
pipeline.to('cuda:0')

cond = Image.open(requests.get("https://d.skis.ltd/nrp/sample-data/lysol.png", stream=True).raw)

result = pipeline(cond, num_inference_steps=75).images[0]

import rembg
result = rembg.remove(result)
result.show()
result.save("output.png")

**Comment by eliphatfs** [comment]
![output](https://github.com/user-attachments/assets/21e7f3da-b4db-431c-85d8-f3885cafafde)
I am running your code changing nothing and here is my result. Maybe there is a problem with the diffusers version?

**Comment by Dumeowmeow** [comment]
> ![output](https://private-user-images.githubusercontent.com/23738781/368752020-21e7f3da-b4db-431c-85d8-f3885cafafde.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY3Mjk1OTAsIm5iZiI6MTcyNjcyOTI5MCwicGF0aCI6Ii8yMzczODc4MS8zNjg3NTIwMjAtMjFlN2YzZGEtYjRkYi00MzFjLTg1ZDgtZjM4ODVjYWZhZmRlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTE5VDA3MDEzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE5ZTA5OWIzYTBmNmRmNmQxZTU5ZWZjNDdiMDRlNzZjMGZhNzNjMTczZmM5NDBhM2UyZmZjOWZjNmQwZGZlYmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.RgUZf2_A1MhkU3s-zrtvKXGW4dUyQ1pOkY2vzW4LBlk) I am running your code changing nothing and here is my result. Maybe there is a problem with the diffusers version?

Thank you! I reinstalled the environment and the problem is solved.
